### PR TITLE
Scaffold archive journal

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: archive-journal.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="container">
+
+  <div class="row">
+
+    <div class="col col-6">
```
Comment (mixcas) [inline archive-journal.php:11]: what is this div?

File: archive-journal.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="container">
+
+  <div class="row">
```
Comment (mixcas) [inline archive-journal.php:9]: only one row? what if articles have diff heights?

Comment (mixcas) [inline archive-journal.php:9]: this div is not even closed…

File: archive-journal.php (hunk 1 above)
Comment (varffs) [inline archive-journal.php:11]: layout. is is the sidebar on the left

File: archive-journal.php (hunk 2 above)
Comment (varffs) [inline archive-journal.php:9]: yes i noticed somewhere that one of the main divs never gets closed :[ I'll fix in this pull.

File: archive-journal.php (hunk 1 above)
Comment (mixcas) [inline archive-journal.php:11]: ok, i just thought it was weird there were blank lines in between :)

**Pull request merged**